src/icons/folderIcons.ts

```diff
@@ -644,6 +644,10 @@ export const folderIcons: FolderTheme[] = [
           'contract-tests',
         ],
       },
+      {
+        name: 'folder-queue',
+        folderNames: ['queue', 'queues', 'bull', 'mq'],
```

Comment:
Hey, I was wondering how the folder names `bull` and `wq` are related to a queue? Are these common terms for queue folder icons?

Reply:
Thank you :)

Reply:
```Bull``` is actually a package name for message queuing, and MQ means **Message Queuing**.
As Bull doesnt require some config _(it's just based on consumers and producers)_, i don't see, _for now_, a reason to make a specific icon for it

There's also ```RabbitMQ```, but i dont see people using it in ```NodeJs```, the most common i could find is Bull

Reply:
Sorry for the late response and thanks for clarifying :)